### PR TITLE
Infra: apply sourcery

### PR DESCRIPTION
**Pull request by Goldziher**
This PR applies sourcery via CLI and fixes all issues it identified. All in all it did a pretty good job, and it only screwed up a few tests.

### CODE REVIEW

**Comment by cofin** [comment]
I agree.  Not bad at all.

**Review by JacobCoffee — APPROVED** [review]

File: tests/docker_service_fixtures.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ async def wait_until_responsive(
     """
     ref = timeit.default_timer()
     now = ref
-    while (now - ref) < timeout:
+    while now - now < timeout:
```
Comment (provinzkraut) [inline tests/docker_service_fixtures.py:40]: This is a bug and seems to have broken our tests.

File: litestar/datastructures/headers.py [hunk 2]
```diff
@@ -429,21 +426,12 @@ def priority(self) -> Tuple[int, int]:
         return quality, specificity
 
     def match(self, other: "MediaTypeHeader") -> bool:
-        # Check parameters first, ignore the weight parameter 'q'.
```
Comment (provinzkraut) [inline litestar/datastructures/headers.py:432]: Comments here should be kept

File: litestar/datastructures/url.py [hunk 3]
```diff
@@ -173,13 +173,14 @@ def from_scope(cls, scope: Scope) -> URL:
         path = scope.get("root_path", "") + scope["path"]
         query_string = scope.get("query_string", b"")
 
-        # # we use iteration here because it's faster, and headers might not yet be cached
```
Comment (provinzkraut) [inline litestar/datastructures/url.py:176]: Comments should be kept

File: tests/docker_service_fixtures.py (hunk 1 above)
Comment (Goldziher) [reply tests/docker_service_fixtures.py:40]: fixed

File: tests/e2e/test_dependency_injection/test_http_handler_dependency_injection.py [hunk 4]
```diff
@@ -90,7 +90,7 @@ def test_function_dependency_injection() -> None:
     )
     def test_function(first: int, second: bool, third: str) -> None:
         assert isinstance(first, int)
-        assert second is False
+        assert not second
```
[the remark this answers is not in the document]
Comment (Goldziher) [reply tests/e2e/test_dependency_injection/test_http_handler_dependency_injection.py:93]: fixed

File: tests/e2e/test_dependency_injection/test_http_handler_dependency_injection.py [hunk 5]
```diff
@@ -63,7 +63,7 @@ class FirstController(Controller):
         def test_method(self, first: int, second: dict, third: bool) -> None:
             assert isinstance(first, int)
             assert isinstance(second, dict)
-            assert third is False
+            assert not third
```
[the remark this answers is not in the document]
Comment (Goldziher) [reply tests/e2e/test_dependency_injection/test_http_handler_dependency_injection.py:66]: fixed

File: docs/examples/contrib/jwt/using_jwt_auth.py [hunk 6]
```diff
@@ -46,12 +46,7 @@ async def retrieve_user_handler(token: Token, connection: "ASGIConnection[Any, A
 @post("/login")
 async def login_handler(data: User) -> Response[User]:
     MOCK_DB[str(data.id)] = data
-    response = jwt_auth.login(identifier=str(data.id), response_body=data)
-
-    # you can do whatever you want to update the response instance here
```
[the remark this answers is not in the document]
Comment (Goldziher) [reply docs/examples/contrib/jwt/using_jwt_auth.py:51]: fixed

File: docs/examples/contrib/jwt/using_oauth2_password_bearer.py [hunk 7]
```diff
@@ -48,26 +48,14 @@ async def retrieve_user_handler(token: "Token", connection: "ASGIConnection[Any,
 @post("/login")
 async def login_handler(request: "Request[Any, Any, Any]", data: "User") -> "Response[OAuth2Login]":
     MOCK_DB[str(data.id)] = data
-    # if we do not define a response body, the login process will return a standard OAuth2 login response.  Note the `Response[OAuth2Login]` return type.
```
[the remark this answers is not in the document]
Comment (Goldziher) [reply docs/examples/contrib/jwt/using_oauth2_password_bearer.py:51]: fixed

File: docs/examples/contrib/jwt/using_oauth2_password_bearer.py [hunk 8]
```diff
@@ -48,26 +48,14 @@ async def retrieve_user_handler(token: "Token", connection: "ASGIConnection[Any,
...
-
-    return response
+    return oauth2_auth.login(identifier=str(data.id))
 
 
 @post("/login_custom")
 async def login_custom_response_handler(data: "User") -> "Response[User]":
     MOCK_DB[str(data.id)] = data
 
-    # If you'd like to define a custom response body, use the `response_body` parameter.  Note the `Response[User]` return type.
```
[the remark this answers is not in the document]
Comment (Goldziher) [reply docs/examples/contrib/jwt/using_oauth2_password_bearer.py:64]: fixed

File: tests/docker_service_fixtures.py (hunk 1 above)
Comment (Goldziher) [reply tests/docker_service_fixtures.py:40]: thats not the reason for the failure, although i reverted this change. The reason is that the event loop closes for some reason.

**Pull request merged**